server.go

```diff
@@ -1193,9 +1194,16 @@ func (s *server) sendPeerMessages(
 	// the queue.
 	var errChans []chan error
 	for _, msg := range msgs {
-		errChan := make(chan error, 1)
+		// If this is not broadcast, create error channels to provide
```

Comment:
👍

Reply:
The goal here is to ensure that `Broadcast` itself can never block? (as now the `writeHandler` won't need to send a sync update back to the caller after each queued message)